src/service/CinemaService.java

```diff
@@ -0,0 +1,178 @@
+package service;
+
+import exception.AlreadyTakenSeatException;
+import exception.NonExistingSeatException;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class CinemaService {
+
+    private static final String availableSeat = "S";
+    private static final String soldSeat = "B";
```

Comment:
The constant naming convention is uppercase with words separated by underscores.
e.g. 
```
private static final String AVAILABLE_SEAT = "S";
```
Can you follow this standard?

Reply:
FIxed